tyr/policies/ec2.py

```diff
@@ -242,8 +276,10 @@
         "sns:Subscribe"
       ],
       "Resource": [
-        "arn:aws:sns:us-east-1:761584570493:s-outpost",
-        "arn:aws:sns:us-east-1:761584570493:alyx3"
+        "arn:aws:sns:us-east-1:761584570493:stage-outpost",
```

Comment:
Yeah, curious as to why you changed the topic ARN?  Looks like the following exist for outpost:

```
t-outpost arn:aws:sns:us-east-1:761584570493:t-outpost
p-outpost arn:aws:sns:us-east-1:761584570493:p-outpost
s-outpost arn:aws:sns:us-east-1:761584570493:s-outpost
```